packages/tasks/src/hardware.ts

```diff
@@ -218,6 +238,18 @@ export const SKUS = {
 				tflops: 45.14,
 				memory: [16, 8],
 			},
+			"RX 6950 XT": {
+				tflops: 47.31,
+				memory: [16],
+			},
+			"RX 6800": {
+				tflops: 41.47,
```

Comment:
```suggestion
				tflops: 32.33,
```

Source: https://www.techpowerup.com/gpu-specs/radeon-rx-6800.c3713